### PR TITLE
Sam492 value of re macro for tpo

### LINKED ISSUE #492: Rate switching under TPO PPA
In the Distributed - Third Party model, there should be an option for commercial customers that finance a behind the meter system with a TPO PPA to have a different rate before and after solar. When solar providers propose to install a system under a PPA for a commercial customer, they often recommend that the customer switch to a solar-friendly rate. Without that there may not be customer savings.

### PR DESCRIPTION
**Pull request by cpaulgilman**
Fix #492 and make macro work for all excess generation compensation (metering) options.

Test files: [value-of-re-macro-test-files.zip](https://github.com/NREL/SAM/files/5973804/value-of-re-macro-test-files.zip)

### CODE REVIEW

**Review by brtietz — APPROVED** [review]

File: deploy/runtime/ui/Utility Rate - Unused.txt [hunk 1]
```diff
@@ -822,7 +822,7 @@ ur_rate_notes
 4
```
Comment (sjanzou) [inline deploy/runtime/ui/Utility Rate - Unused.txt:822]: Should this be merged?

File: deploy/runtime/startup.lk [hunk 2]
```diff
@@ -263,15 +263,38 @@ function setup_residential_pages( mode ) // mode determines whether or not the b
   setup_elec_load_page( mode );
 }
 
-function setup_thirdparty_pages( load ) // FOR NOW, THIRD PARTY OWNERSHIP MODEL IS ONLY VALID FOR PVWATTS AND PV DETAILED and generic system!!
+function setup_thirdparty_pages( load ) // Third party host
 {
   setting( { 'analysis_period_var'='cf_length' } );
   addpage( [[ 'Financial Third Party Ownership']],
-           { 'sidebar'='Financial Parameters', 'help'='fin_thirdparty'} );
+           { 'sidebar'='Financial Parameters', 'help'='fin_tpo_host'} );
```
Comment (sjanzou) [inline deploy/runtime/startup.lk:270]: Thanks for breaking out the TPO models - any additional variables added or removed based on new set of stacked pages for patching?

**Review by sjanzou — APPROVED** [review]
Please check the questions in my comments and why Travis is failing... Thanks!

File: deploy/runtime/startup.lk (hunk 2 above)
Comment (cpaulgilman) [reply deploy/runtime/startup.lk:270]: This is only an update to the 'help' parameter of addpage() so it points to the correct topic in Help. It does not change the UI pages.

**Pull request merged**